microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -34,23 +34,31 @@ When fixing a bug, you often want to try out some changes to your JavaScript cod
 
```

Comment:
change 25 from:
This article covers how to use the debugger in Microsoft Edge DevTools, including how to set a line-of-code breakpoint. To set other types of breakpoints, see [Pause your code with breakpoints](breakpoints.md).
to:
This article covers how to use the debugger features in Microsoft Edge DevTools, including how to set a line-of-code breakpoint. To set other types of breakpoints, see [Pause your code with breakpoints](breakpoints.md).

Add "features", to give sharper contrast between the 3 articles discussed at top